yaml/kubernetes/best-practice/no-fractional-cpu-limits.yaml

```diff
@@ -16,10 +16,4 @@ rules:
     even though most nodes can handle processes
     alternating between using 100% of the CPU.
   severity: WARNING
-  paths:
```

Comment:
Is `.k8s` a common extension for these configs?

The `.yml` and `.yaml` should be covered after https://github.com/returntocorp/semgrep/pull/2877.

Reply:
I think it is; k8s is kubernetes configs I believe

Reply:
Related issue: https://github.com/returntocorp/semgrep/issues/2837

Reply:
I've never seen `.k8s` used in the wild so I'd remove it.

Reply:
@sim1 have you ever seen manifests use a `.k8s` extension?

Reply:
no, this is the first I hear of a .k8s file extension @underyx